addons/calendar/models/calendar_attendee.py

```diff
@@ -140,7 +140,6 @@ def _send_mail_to_attendees(self, mail_template, force_send=False):
                     email_layout_xmlid='mail.mail_notification_light',
                     attachment_ids=attachment_values,
                     force_send=force_send,
-                    message_type='auto_comment',
```

Comment:
`"auto_comment"` is necessary to notify users of replies to this message (internal messages create note responses by default, and notifications are always internal messages), that's the reason that message type was added.

Instead you want to change the call to send individual emails, such that it's never logged in the chatter. If you want to keep the chatter log, you should call `_message_log`, which will just log the message and notify no one.

Reply:
Which change is the right one to solve this actual incorrect use case you propose?

The proposed change (remove `message_type='auto_comment'`) is the same as what is already in [saas-16.2](https://github.com/odoo/odoo/blob/saas-16.2/addons/calendar/models/calendar_attendee.py), [saas-16.3](https://github.com/odoo/odoo/blob/saas-16.3/addons/calendar/models/calendar_attendee.py), [saas-16.4](https://github.com/odoo/odoo/blob/saas-16.4/addons/calendar/models/calendar_attendee.py), [17.0](https://github.com/odoo/odoo/blob/17.0/addons/calendar/models/calendar_attendee.py#L170), and [master](https://github.com/odoo/odoo/blob/master/addons/calendar/models/calendar_attendee.py#L170) (only [16.0](https://github.com/odoo/odoo/blob/16.0/addons/calendar/models/calendar_attendee.py) and [saas-16.1](https://github.com/odoo/odoo/blob/saas-16.1/addons/calendar/models/calendar_attendee.py) have this problem).